README.md

```diff
@@ -1,7 +1,7 @@
 # sbt-coveralls
 
 [![Join the chat at https://gitter.im/scoverage/sbt-coveralls](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/scoverage/sbt-coveralls?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[<img src="https://img.shields.io/maven-central/v/sbt-coveralls/sbt-coveralls.svg?label=latest%20release"/>](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22sbt-coveralls)
+[<img src="https://img.shields.io/badge/latest%20release-1.2.2-green.svg"/>](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22sbt-coveralls)
```

Comment:
this looks like a hardcoded version
I opened #100 some time ago, where correct version resolved properly using `maven-badges.herokuapp.com`

Reply:
cool, TIL about that heroku, thanks

seems like #100 is better than this, will close

should we see if you can be added to scoverage org to help reviewing/merging?

Reply:
i wouldn't mind, to be added to scoverage (or any other org)
I can definitely help reviewing/merging, but can't commit to certain level of activity though

Reply:
cool. seems like @gslowikowski has been active around here recently, maybe he can chime in about giving you the commit bit…